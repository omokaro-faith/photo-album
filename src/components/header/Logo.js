import React from 'react';
import { Link } from 'react-router-dom';
import '../../styles/components/header/_logo.css';

const LogoPage = () => (
  <div className='logo__wrapper'>
    <Link to='/' className='logo__wrapper-item'> AlaBin </Link>
  </div>
)

 export default LogoPage; 