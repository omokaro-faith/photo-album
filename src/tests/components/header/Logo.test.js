import React from 'react';
import { shallow } from 'enzyme';
import Logo from '../../../components/header/Logo';

describe('Logo', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Logo />);
  });

  afterEach(() => { wrapper = null; });

  
  describe('renders', () => {
    it('should exist', () => {
      expect(wrapper.exists()).toBe(true);
    });

    it('should not have any regressions', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
