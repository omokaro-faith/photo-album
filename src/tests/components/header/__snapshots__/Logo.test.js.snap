// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders should not have any regressions 1`] = `ShallowWrapper {}`;
